### ISSUE TITLE
[Feature] TTTTimeIntervalFormatter Rounding

### ISSUE THREAD

**Issue by johnboiles** [FormatterKit/FormatterKit#196]
Currently, TTTTimeIntervalFormatter doesn't do any rounding. Take for example the following.

``` objc
TTTTimeIntervalFormatter  *timeIntervalFormatter = [TTTTimeIntervalFormatter new];
[timeIntervalFormatter setUsesIdiomaticDeicticExpressions:YES];

// Format date for one second less than 2 days ago (2 * 24 * 60 * 60 - 1)
NSLog(@"%@", [timeIntervalFormatter stringForTimeInterval:-172799]); // "yesterday"
// Format date for one second less than 2 weeks ago (14 * 24 * 60 * 60 - 1)
NSLog(@"%@", [timeIntervalFormatter stringForTimeInterval:-1209599]); // "last week"
// Format date for one second less than three days ago ( 3 * 24 * 60 * 60 - 1)
NSLog(@"%@", [timeIntervalFormatter stringForTimeInterval:-259199]); // "2 days ago"
// Format date for one second less than three hours ago ( 3 * 60 * 60 - 1)
NSLog(@"%@", [timeIntervalFormatter stringForTimeInterval:-10799]); // "2 hours ago"
```

I've gotten feedback that this is confusing in my app. It's surprising to see "yesterday" for something that happened 47 hours ago. Likewise, if it's Monday, and something happened on Friday, it's surprising to show "2 days ago".

**Comment by mciarlo** [comment]
Would love this as well. Currently seeing dates that occur two days after today show up as "Tomorrow"

**Comment by mattt** [comment]
This is fixed by #163.
